src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>PowerShell.Core.Instrumentation</id>
+    <version>6.0.0-beta.9</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>PowerShell Core ETW resource binary</description>
+    <copyright>Copyright 2017 Microsoft</copyright>
```

Comment:
We use [the string ](https://github.com/PowerShell/PowerShell/blob/master/PowerShell.Common.props#L94) `(c) Microsoft Corporation. All rights reserved.`

Also should we start all XML tags with capitalization?

Reply:
I did a survey of all nuspec files in my package cache, and they all use lowercase for the xml elements.

Reply:
Fixed the copyright text